src/main/comments/print.js

```diff
@@ -203,7 +203,7 @@ function printComments(path, doc, options) {
   return label(
     // Propagate object labels so that the printing logic for ancestor nodes
     // could easily check them.
-    typeof doc.label === "object" && { commented: true, ...doc.label },
+    doc.label,
```

Comment:
@thorn0 `commented` seems not used. Not sure why added in #13532. I think it's safe to use directly.